src/rest/api/plant.js

```diff
@@ -2,7 +2,7 @@ import Client from 'rest/baseClient'
 
 const http = new Client()
 
-export const getAll = async () => {
+export const getAllPlant = async () => {
```

Comment:
Use the name `getAll` instead of this one. Because this API in `plant.js` not other files. Shouldn't use redundant sentences, give shorter names